server.js

```diff
@@ -124,6 +124,7 @@ async function work(body) {
     findPotentialReviewers: true,
     actions: ['opened'],
     skipAlreadyAssignedPR: false,
+    reviewerAssignment: null,
```

Comment:
the other options seems to be verbs. Could you rename it to `assignToReviewer` or something along those lines